rest_framework/templatetags/rest_framework.py

```diff
@@ -49,20 +49,26 @@ def with_location(fields, location):
     ]
 
 
+
+# TOOD: Where to pull these helpers?
+def render_to_form(schema):
+    assert False, "Can't rely on coreschema encodings here."
+
+
 @register.simple_tag
 def form_for_link(link):
-    import coreschema
+    from coreapi import typesys
     properties = OrderedDict([
-        (field.name, field.schema or coreschema.String())
+        (field.name, field.schema or typesys.String())
         for field in link.fields
     ])
     required = [
         field.name
         for field in link.fields
         if field.required
     ]
-    schema = coreschema.Object(properties=properties, required=required)
-    return mark_safe(coreschema.render_to_form(schema))
+    schema = typesys.Object(properties=properties, required=required)
+    return mark_safe(render_to_form(schema))
```

Comment:
This is the missing helper. You'll see I've just blocked in an `assert False` above for the moment.